cmd/nerdctl/stats.go

```diff
@@ -306,6 +306,9 @@ func statsAction(cmd *cobra.Command, args []string) error {
 	ticker := time.NewTicker(500 * time.Millisecond)
 	defer ticker.Stop()
 
+	// firstTick is for creating distant CPU readings.
+	// firstTick stats are not displayed.
+	var firstTick = true
 	for range ticker.C {
 		cleanScreen()
```

Comment:
```suggestion
		cleanScreen()
		if fistTick && noStream {
		        firstTick = false
		        continue
		}
```

Reply:
Nope , because this fixes stream and no-stream  and we have to render at least one entry to make distant CPU https://github.com/containerd/nerdctl/pull/1233/files#diff-8541fa08e70d092bb039071593918dac552c3cbf1088d15e71d0f6117ad4b249R329